### PR TITLE
Python packages version upgrades.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
Just package upgrades, merging.

File: {{ cookiecutter.project_name }}/requirements.txt [hunk 1]
```diff
@@ -53,37 +55,37 @@ django-csp==2.0.3
...
 
 # sha256: F2PRVkET9euJ33eHmo0yEyc8TX_5Pcs3owcM3ww0_Xw
 ipdb==0.8.1
 
-# sha256: Ld8Y2jsGIfpD_uS3KQ2grnibRvuJkniorMzaGVxJeac
-# sha256: Gm5RMMK0LS3jAWk8KZ94zEvTUB54thDAjkXvxw4rURQ
-Sphinx==1.3.1
+# sha256: a2nyehEIU2b-2tJbs8OP5wY5cSp-SiNKJX6irR-PZqE
+# sha256: t9Ezu0mQ0BCirZNMMZtS2KIVbLBJFIT14qVYYZvJrgQ
+Sphinx==1.3.5
```
Comment (peterbe) [inline {{ cookiecutter.project_name }}/requirements.txt]: Why is this in here? Don't we have `docs/requirements.txt` for that?

Comment (glogiotatidis) [inline {{ cookiecutter.project_name }}/requirements.txt]: Yes, and this pull request was before #114

Comment (peterbe) [inline {{ cookiecutter.project_name }}/requirements.txt]: /me good catch :)

**Pull request merged**